src/core/parser.ml

```diff
@@ -609,6 +611,9 @@ let parser statement =
 let parser proof =
   ts:tactic* e:proof_end -> (ts, Pos.in_pos _loc_e e)
 
+let parser keyword_proof =
+  _proof_ ts_pe:proof -> ts_pe
```

Comment:
These two parsers could be merged into a single one:
```
let parser proof =
  _proof_ ts:tactic* e:proof_end -> (ts, Pos.in_pos _loc_e e)
```

Reply:
fixed in 7ff9ab7